server/route.go

```diff
@@ -1163,7 +1161,7 @@ func (s *Server) createRoute(conn net.Conn, rURL *url.URL) *client {
 	// Do final client initialization
 
 	// Initialize the per-account cache.
-	c.in.pacache = make(map[string]*perAccountCache, maxPerAccountCacheSize)
+	c.in.pacache = make(map[string]*perAccountCache)
```

Comment:
Should we do the same for gateways? There is the exact same code there..

Reply:
Yes separate PR though..